processors/base/helpers/__init__.py

```diff
@@ -266,3 +277,56 @@ def safe_prepend(prepend_string, string):
         string = '%s%s' % (prepend_string, string)
 
     return string
+
+
+def get_canonical_location_name(location):
+    """Find the canonical location name according to the
+    passed entry
+
+    Args:
+        location (str): the location to be normalized
+    """
+
+    # This auxiliar file is provided by the dataset repository of github.com and
+    # can be checked on its latest version on the project website
+    # See: https://github.com/datasets/country-codes/blob/master/data/country-codes.csv
+    #
+    # To create e new entry (or update an existing one), add a new line (or find the
+    # existing one) and fill in the informations according to the header (line 1) order,
+    # separating fields using comma
+    CSV_PATH = 'data/countries.csv'
+    ALPHA_3_HEADER = 'ISO3166-1-Alpha-3'
+    DISTANCE_SCORER = fuzz.token_sort_ratio
+    SCORE_INDEX = 1
```

Comment:
There's no need to add the `ALPHA_3_HEADER`, as it's used only once and the column name is as expressive as the constant name.